api/hypershift/v1beta1/aws.go

```diff
@@ -38,16 +38,17 @@ type AWSNodePoolPlatform struct {
 	RootVolume *Volume `json:"rootVolume,omitempty"`
 
 	// resourceTags is an optional list of additional tags to apply to AWS node
-	// instances.
+	// instances. Changes to this field will be propagated in-place to AWS node instances.
```

Comment:
and the default security group? and ec2 enis?

Reply:
just realised this is the nodepool so no default security group, but it does impact the enis and specific nodepool sgs, no? can we clarify that if true

Reply:
afaic, CAPA doesn't reconcile tags on ENI and securityGroups. I creates the tags on the ENI on creation only.

Reply:
updated